carbonmark-api/README.md

```diff
@@ -31,38 +31,27 @@ You will need to be authenticated to the following in order to run the project:
 - run from root
 
 ```sh
-vercel  link
+vercel link
 ```
 
 ### Environment Variables
 
-The vercel config is where all environment variables required for the project are defined and can be pulled down via the `vercel-cli`.
+Some important environment variables are stored in vercel and need to be downloaded to your machine.
 
-Using the below command, the newest version of vercel cli will create a `.vercel` folder in the root and install the env variables in `.vercel/.env.development.local`.
-
-Previous versions created and stored in an `.env.local` in the root.
-
-Ensure you have updated vercel cli to the latest version (31.0.4 as of this doc)
-
-### **Note:** You will need to manually remove the beginning and trailing quotation marks on the `FIREBASE_ADMIN_CERT` env var in `.vercel/env.development.local` otherwise you will receive a _SyntaxError: Unexpected token_ error when trying to run the project
-
-- run from root
+### **Note:** You will need to manually remove the beginning and trailing quotation marks on the `FIREBASE_ADMIN_CERT` env var in `.env.local` otherwise you will receive a _SyntaxError: Unexpected token_ error when trying to run the project
```

Comment:
Removed thanks!

On my laptop I ran into issues here, I couldn't get local firebase requests to work (firestore API was rejecting the cert). So it might be an expired cert, the wrong cert, or I did something wrong.

Site note: I have never had problems storing the entire admin cert JSON as a string in vercel, the reason this quotation mark issue arises is because local builds rely on `dotenv`. I'll copy paste this comment as a note to add here, if we get around to cleaning up env vars in a PR: https://github.com/KlimaDAO/klimadao/issues/1281